File: app/src/main/java/co/com/ceiba/mobile/pruebadeingreso/user/model/Address.java
package co.com.ceiba.mobile.pruebadeingreso.user.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * <p>
 *
 * </p>
 *
 * Create By Zorayda 11/3/2020
 * @author Mona15 2020
 * @version 1.0
 */
public class Address {
    @SerializedName("street")
    @Expose
    String street;
    @SerializedName("suite")
    @Expose
    String suite;
    @SerializedName("city")
    @Expose
    String city;
    @SerializedName("zipcode")
    @Expose
    String zipcode;
    @SerializedName("geo")
    @Expose
    Geo geo;
}

File: app/src/main/java/co/com/ceiba/mobile/pruebadeingreso/users/model/Address.kt
package co.com.ceiba.mobile.pruebadeingreso.users.model

import com.google.gson.annotations.Expose

import com.google.gson.annotations.SerializedName

class Address {
    @SerializedName("street")
    @Expose
    private val street: String? = null

    @SerializedName("suite")
    @Expose
    private val suite: String? = null

    @SerializedName("city")
    @Expose
    private val city: String? = null

    @SerializedName("zipcode")
    @Expose
    private val zipcode: String? = null

    @SerializedName("geo")
    @Expose
    private val geo: Geo? = null
}
